i18n/mk.yaml

```
@@ -0,0 +1,93 @@
+# General
+- id: read_more
+  translation: "Читај повеќе…"
+
+- id: menu_label
+  translation: "Мени"
+
+- id: sidemenu_title
+  translation: "Мени"
+
+# Post meta
+- id: meta_lastmod
+  translation: "Последна промена"
+
+- id: meta_translations
+  translation: "Преводи"
+
+# Table of Contents
+- id: toc_title
+  translation: "Содржина"
+
+# Post nav
+- id: post_nav_prev
+  translation: "Претходна"
+
+- id: post_nav_next
+  translation: "Следна"
+
+# Authorbox
+- id: authorbox_name
+  translation: "За {{ .Count }}"
+
+# Sidebar
+- id: sidebar_warning
+  translation: "ВНИМАНИЕ"
+
+- id: sidebar_recommendation
+  translation: "Ве молам, активирајте најмалку еден страничен елемент."
+
+# Search widget
+- id: search_placeholder
+  translation: "Пребарувај..."
+
+# Languages widget
+- id: languages_title
+  translation: "Јазици"
+
+# Categories widget
+- id: categories_title
+  translation: "Категории"
+
+# Recent Posts widget
+- id: recent_title
+  translation: "Скорешни написи"
+
+# Social widget
+- id: social_title
+  translation: "Социјални"
+
+# Tags List widget
+- id: tags_title
+  translation: "Ознаки"
+
+# Footer
+- id: footer_credits
+  translation:
+    "Направено со <a href=\"https://gohugo.io/\" rel=\"nofollow noopener\" target=\"_blank\">Хуго (Hugo)</a> и \
+    тема <a href=\"https://github.com/Vimux/Mainroad/\" rel=\"nofollow noopener\" target=\"_blank\">Главен пат (Mainroad)</a>."
```

```suggestion
    "Направено со <a href=\"https://gohugo.io/\" rel=\"nofollow noopener\" target=\"_blank\">Hugo</a> и тема \
    <a href=\"https://github.com/Vimux/Mainroad/\" rel=\"nofollow noopener\" target=\"_blank\">Mainroad</a>."
```

1. Indentation
1. Don't translate "Hugo" and "Mainroad". I don't know how the translation sounds after my suggestion. Check this, please.

---

Yes, my mistake. Names should not be translated, I though the meaning behind Mainroad should be stressed. 

Also it is good to know that since Macedonian uses a different script (Cyrillic), according to the grammar rules, it should be at least transliterated: Hugo = Хуго, Mainroad = Мејнроад. 

But it's fine.